cluster/images/hyperkube/master-multi.json

```diff
@@ -21,12 +29,19 @@
       "command": [
               "/hyperkube",
               "apiserver",
+              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
Correct, is it deisred ot phase out `SecurityContextDeny` everywhere.

On Fri, Nov 20, 2015 at 10:49 PM, Emma He <notifications@github.com> wrote:

> In cluster/images/hyperkube/master-multi.json
> <https://github.com/kubernetes/kubernetes/pull/17213#discussion_r45540995>
> :
>
> > @@ -21,12 +29,19 @@
> >        "command": [
> >                "/hyperkube",
> >                "apiserver",
> > +              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
>
> Ss of today, we have
>
> type SecurityContext struct {
>     Capabilities *Capabilities `json:"capabilities,omitempty"`
>     Privileged *bool `json:"privileged,omitempty"`
>     SELinuxOptions *SELinuxOptions `json:"seLinuxOptions,omitempty"`
>     RunAsUser *int64 `json:"runAsUser,omitempty"`
>     RunAsNonRoot *bool `json:"runAsNonRoot,omitempty"`
> }
>
> So it's quite safe to phase out SecurityContextDeny IIUC
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17213/files#r45540995>.
>